docs/build-with-powerloom/devnet/getting-started.md

```diff
@@ -76,26 +86,170 @@ Clone the repository against the respective branch (main by default). Open the t
 git clone https://github.com/PowerLoom/deploy.git --single-branch powerloom_deploy --branch devnet && cd powerloom_deploy
 ```
 
-#### Step 2: Fork the Snapshotter Computes and Snapshotter Config Repositories
+#### Step 2: Navigate to the Repository Directory:
+Change to the cloned repository's directory:
+
+```bash    
+    cd powerloom-deploy
+```
+
+#### Step 3: Configure the environment variables
+
+1. In the deploy repo's directory, create a new file named `.env`.
+
+2. Copy the contents of [`env.example`](https://github.com/PowerLoom/deploy/blob/devnet/env.example) to `.env`.
+
+3. Fill in the required variables in the `.env` file:
+
+#### During the creation of the .env file, add the following details: 
+- `SOURCE_RPC_URL`: The URL for Source RPC (Local node/Infura/Alchemy) service.
+- `SIGNER_ACCOUNT_ADDRESS`: The address of the signer account. This should be an unused "burner" address that does not need to have any token balance. You can create a new burner wallet from [our tool](https://devnet-mint.powerloom.dev/burner). During later steps, this address will be registered against the `SLOT_ID` on the `PROTOCOL_STATE_CONTRACT` that is set in this `.env`.
+- `SIGNER_ACCOUNT_PRIVATE_KEY`: The private key corresponding to the burner wallet address
+- `SLOT_ID`: Enter your Devnet Slot ID. [You can find the slotID by visiting the [Devnet Dashboard](https://devnet-mint.powerloom.dev/dashboard), or on the [Sepolia block explorer](https://sepolia.etherscan.io/) by searching for the transaction hash of your node mint]
+- `PROTOCOL_STATE_CONTRACT`: Choose the contract address for the protocol state with respect to the Epoch size.
 
-Our system leverages the Git submodule architecture to seamlessly manage these components. To dive deeper into how these elements integrate and function within our larger system, check out the [architecture documentation](https://docs.powerloom.io/docs/build-with-powerloom/snapshotter-node/architecture).
+:::note
+Currently, there are two variations of Protocol State Contracts available. One where [Epoch](../../Protocol/Specifications/Epoch.md) size is set to 1 and other where epoch size is set to 10 blocks. 
 
-To optimize the development process, we recommend forking the templates for:
+You can find the latest Epoch Contracts in the [example.env](https://github.com/PowerLoom/deploy/blob/devnet/env.example) file. 
 
-- Snapshotter Configs: https://github.com/PowerLoom/snapshotter-configs
-- Snapshotter Computes: https://github.com/PowerLoom/snapshotter-computes
+This should allow developers to build and experiment with a variety of use cases. If you have any custom needs, reach out to the [team](https://discord.com/invite/powerloom)
+:::
+
+**Already Given:**  
```

Comment:
Pre-supplied is a better phrase.

Also should be noted that unless it is a customized need or instructed by us, these need not be changed or modified.